doc/nat/nat_design_spec.md

```diff
@@ -1520,31 +1529,31 @@ The Unit test case one-liners are as below:
 | 43 | Error log when errors are seen in receiving netlink messages from conntrack by NatSyncd (for the dynamic NAPT entry notifications from the kernel).                                                                                   |
-| 44 | Error messages are logged if NatMrgd gets errors writing to APP_DB.                                                                                                                                                                    |
-| 45 | Error messages are logged if NatOrch gets errors when writing to ASIC_DB.                                                                                                                                                              |
-| 46 | Error messages are logged if Syncd gets errors when writing to COUNTERS_DB.                                                                                                                                                            |
-| 47 | Verify that the received NAT source miss and destination miss packets are 'trapped' to CPU on the right COS queue (Queue 3). The queue assignment should be lower than protocol queues and higher than broadcast packets.             |
-| 48 | Verify that the NAT source miss and destination miss packets are 'rate limited' to CPU (600pps).                                                                                                                                      |
-| 49 | Verify that dynamic NAT entries are learnt even during a BCAST/MCAST storm (at line rate).                                                                                                                                            |
-| 50 | Verify the tracing of natmgrd at different log levels.                                                                                                                                                                                 |
-| 51 | Verify the tracing of natorch at different log levels.                                                                                                                                                                                |
-| 52 | Execute the debug dump commands for dumping the internal operational data/state of NatOrch and NatMgrd.                                                                                                                               |
-| 53 | Verify NAT happens on ICMP traffic like ping, traceroute traffic.                                                                                                                               |
-| 54 | Verify NAT happens on TCP traffic like ssh, telnet and UDP traffic like TFTP.                                                                                                                               |
```

Comment:
Could we please add and verify test cases when static NAT entries are deleted, as well as the ACLs are changed for both static and dynamic NAT?

Reply:
Done.